src/test/java/service/impl/FileServiceImplTest.java

```diff
@@ -0,0 +1,41 @@
+package service.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import java.util.List;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import service.FileService;
+
+public class FileServiceImplTest {
```

Comment:
for my opinion, here need to use Interface segregation principle, but you may be not agree, so your need to use reader and writer in different classes and have different test classes